### PR TITLE
FontAwesome 5.6.1, fix XML docs, unit tests & concurrency bug fix.

### PR DESCRIPTION
**Pull request by johanbenschop**
Merry Christmas! The primary change in this pull request is the update of FontAwesome to version 5.6.1.

Also:
- Improved the generator which now directly pulls from GitHub and updates the files in one fell swoop (given FontAweomse doesn't change their folder structure).
- Now including the DocumentationFile inside the NuGet package so consumers get the documentation of the enums.
- Improved the unit tests to use XUnits MemerData functionality to generate tests per value in an enum. 
- Two types of tests, one for embedded and one for custom fonts. Total 4308 tests ~50 seconds to run them. 
- Tests now count the unique colors per image to verify the icon could be found.
- The tests give a code coverage of 55.73%.
- While making these tests I got random exceptions with something about an error in GDI+ when running tests in parallel with multiple font styles. Solved this by adding a MemoryCache to the LoadFontFamily methods in the FontThumbnailService.
- While debugging I also improved the code, like adding in more using statements for disposing of objects.

I look forward to feedback (if any).

### CODE REVIEW

File: src/Geta.Epi.FontThumbnail.Tests/Geta.Epi.FontThumbnail.Tests.csproj [hunk 1]
```diff
@@ -5,6 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="EPiServer.CMS.UI.Core" Version="11.1.0" />
```
Comment (degborta) [inline src/Geta.Epi.FontThumbnail.Tests/Geta.Epi.FontThumbnail.Tests.csproj:8]:
```suggestion
     <PackageReference Include="EPiServer.CMS.UI.Core" Version="11.2.1" />
```

**Review by degborta — CHANGES_REQUESTED** [review]
Merry christmas @johanbenschop  and thanks for the contribution. It looks really nice! Only stumbled on one version conflict that I added a suggestion for. Take a look and get back with your comments.

File: src/Geta.Epi.FontThumbnail.Tests/Geta.Epi.FontThumbnail.Tests.csproj (hunk 1 above)
Comment (johanbenschop) [reply src/Geta.Epi.FontThumbnail.Tests/Geta.Epi.FontThumbnail.Tests.csproj:8]: Aye. I just noticed that the Episerver packages in the main project are set to version 11.2.1 except for `EPiServer.CMS.UI.Core`. But shouldn't they all be set to the same version as the minimum version of the Episerver dependancy in the .nuspec file, 11.1.0 in this case?

Comment (degborta) [reply src/Geta.Epi.FontThumbnail.Tests/Geta.Epi.FontThumbnail.Tests.csproj:8]: Yeah, you are ofcourse right. Guess I was too fast to fix the version conflict :)

Can you commit that to the PR?

Comment (johanbenschop) [reply src/Geta.Epi.FontThumbnail.Tests/Geta.Epi.FontThumbnail.Tests.csproj:8]: Committed a fix to this PR.

**Review by degborta — APPROVED** [review]

**Pull request merged**